### PR TITLE
new segmentation algorithm, faster and more accurate

### PR DESCRIPTION
**Pull request by ahuizxc**

### CODE REVIEW

File: object_analytics_node/src/model/object_utils.cpp [hunk 1]
```diff
@@ -43,35 +43,16 @@ void ObjectUtils::fill3DObjects(const ObjectsInBoxes3D::ConstSharedPtr& objects_
...
-      max = area;
-      max_it = it;
-    }
-
-    if (max <= 0)
+    return;
+  }
+  else
+  {
+    for (size_t i=0;i<objects2d.size();i++)
```
Comment (yechun1) [inline object_analytics_node/src/model/object_utils.cpp]: please insert blank space after first ";"

**Comment by yechun1** [comment]
CI build failed, please change the code and pass ament test.

**Comment by ahuizxc** [comment]
好的

**Comment by yechun1** [comment]
I'll enable ament_cppcheck to check code style issue, the code is lgtm, I'll merge it.

**Pull request merged**